lib/sacastats_web/views/character_view.ex

```diff
@@ -112,15 +120,63 @@ defmodule SacaStatsWeb.CharacterView do
     """
   end
 
-  defp build_event_log_item(assigns, %Events.VehicleDestroy{} = vd, %Session{} = _session) do
+  defp build_event_log_item(assigns, %VehicleDestroy{} = vd, %Session{}, character_map) do
+    character_identifier = get_character_name(assigns, character_map, vd.character_id)
+    attacker_identifier = get_character_name(assigns, character_map, vd.attacker_character_id)
+
     ~H"""
     <li>
-      <%= vd.attacker_character_id %> destroyed <%= vd.character_id %>'s <%= SacaStats.vehicles()[vd.vehicle_id]["name"] %>
+      <%= attacker_identifier %> destroyed <%= character_identifier %>'s <%= SacaStats.vehicles()[vd.vehicle_id]["name"] %>
       with <%= SacaStats.weapons()[vd.attacker_weapon_id]["name"] %> (<%= vd.attacker_weapon_id %>)
       <%= vd.attacker_vehicle_id != 0 && " while in a #{SacaStats.vehicles()[vd.attacker_vehicle_id]["name"]}" || "" %>
       -
       <%= SacaStatsWeb.CharacterView.prettify_timestamp(assigns, vd.timestamp) %>
     </li>
     """
   end
+
+  defp build_event_log_item(
+         assigns,
+         %GainExperience{experience_id: id, character_id: character_id} = ge,
+         %Session{character_id: character_id},
+         character_map
+       )
+       when is_assist_xp(id) do
+    other_identifier = get_character_name(assigns, character_map, ge.other_id)
+    character_identifier = get_character_name(assigns, character_map, character_id)
+
+    ~H"""
+    <li>
+      <%= character_identifier %> assisted in killing <%= other_identifier %> (<%= ge.amount %>%)
```

Comment:
I think this is kind of right, but I do tend to get some outliers (happened like 6 times in a 3 1/2 hour session) like this these with an over 100% assist. That just mean I hit as they heal and don't get the kill?
[dndmackey](http://lvh.me:4000/character/dndmackey) assisted in killing [1477472447154](http://lvh.me:4000/character/1477472447154) (Character Search Failed) (256%) - 2022-08-29 05:25:24
[dndmackey](http://lvh.me:4000/character/dndmackey) assisted in killing [1477472421858](http://lvh.me:4000/character/1477472421858) (Character Search Failed) (230%) - 2022-08-29 05:20:32
[dndmackey](http://lvh.me:4000/character/dndmackey) assisted in killing [NotInTheEyeThatBurns](http://lvh.me:4000/character/NotInTheEyeThatBurns) (145%) - 2022-08-29 05:05:17
[dndmackey](http://lvh.me:4000/character/dndmackey) assisted in killing [TheDawn100](http://lvh.me:4000/character/TheDawn100) (201%) - 2022-08-29 05:05:11

To name a few

Reply:
Yeah this is a good observation - I didn't think about them healing to throw off the percent. It could also be affected by XP multipliers/boosts. Do you think we should take out the percentage entirely (at least for now)?

Reply:
Ah, okay. Yeah, I would probably leave it off for now, but if you can find a way to keep it, that would be cool. I always like seeing what type of damage I did to them.

Reply:
Okay I think I'll remove that for now - maybe it'll be something we revisit for 2.0